### DISCUSSION TITLE
Deferred function call

### DISCUSSION THREAD

**Discussion by harshmangalam** [go101/go101#237] [Q&A] [answered]
In deferred function call topic
go101 say that :- deferred call  is pushed into a deferred call queue 
in go doc :-  deferred function call is pushed into deferred call stack

**Comment by zigo101** [comment, marked as answer]
@harshmangalam 
Sorry, I just noticed this discussion.

Go 101 also ever used the `stack` word before.
It is just a name. The meaning is the same as the `stack` used in the go doc.
I changed the word to `queue` to make it more distinguishable from the normal function call stack.
